modules/ROOT/pages/installation/requirements.adoc

```diff
@@ -108,9 +108,19 @@ For cloud environments, and server-based, on-premise environments:
 | *Windows Server 2016, 2019, 2022*                | OracleJDK 17, ZuluJDK 17
 |===
 
+label:deprecated[Deprecated in 5.26]
+Starting from Neo4j 5.26 LTS, deploying the database on the following operating systems is deprecated:
+
+* MacOs 11, 12
+* Windows Desktop 10
+* Amazon Linux 2022 AMI
+* Unbuntu Server 16.04, 18.04, 20.04
+* Windows Server 2016, 2019
+
 [NOTE]
 ====
-From version 5.14 onwards, Neo4j also supports JDK 21 on all Operating Systems.
+The Neo4j 5.26 LTS is the last version that uses the Java SE 17 Platform.
```

Comment:
I think this note is good but should be moved to the Java section.

Reply:
good